Desktop/qquick/datasetview.cpp

```diff
@@ -1295,7 +1295,7 @@ QString DataSetView::columnInsertBefore(int col, bool computed, bool R)
 
 	_model->insertColumn(col, computed, R);
 
-	return "";
+	return tq(DataSetPackage::pkg()->dataSet()->column(col)->name());
```

Comment:
This will go wrong as soon as we start sorting or filtering columns, but its probably not the only place